frontend/src/components/activity/ScheduleEntry.vue

```diff
@@ -314,6 +316,7 @@ export default {
       categoryChangeState: null,
       loading: true,
       isPaperDisplaySize: true,
+      isLocalPaperDisplaySize: true,
```

Comment:
Why is `isLocalPaperDisplaySize` needed?

Reply:
It is to prevent ugly layout jumps. I want to change the responsive layout before the width is transitioned, because otherwise the resize event is executed only after the max width is done transitioning.